lib/webdriverio.js

```diff
@@ -230,6 +234,14 @@ let WebdriverIO = function (args, modifier) {
             })
     }
 
+    function mkRequestHanlderDecorator (opts) {
```

Comment:
No `mkRequestHandlerDecorator` rather `mkRequestHanlderDecorator` and yes `makeRequestHandlerDecorator` would be even better.